### PR TITLE
added windows 10 template

### PR DESCRIPTION
**Pull request by edewit**
fixing issue [FH-1700](https://issues.jboss.org/browse/FH-1700)

### CODE REVIEW

**Comment by edewit** [comment]
build is failing because it's looking for an image that is included in the PR, can that be changed?

**Comment by matzew** [comment]
is this ready to merge? or still issues ?

**Comment by jasonmadigan** [comment]
Build is failing with a 404 fetching `https://raw.githubusercontent.com/feedhenry/template-config/master/screenshots/apps/blank_native_windows_universal/2.png` - if there's a PR open with that change, that'll have to be merged before we can run the build job here (which pulls down all templates and packages them up).

**Comment by edewit** [comment]
@jasonmadigan so the build that checks the PR fails because it can't find an image that the PR is including. It's a catch 22

File: global.json [hunk 1]
```diff
@@ -611,6 +611,23 @@
         ]
       },
       {
+        "id": "blank_native_windows10",
+        "name": "Blank Native Universal Windows 10 App",
+        "type": "client_native_windowsuniversal",
+        "priority": 0.44,
+        "description": "Blank Hello World Web App",
+        "repoUrl": "git@github.com:feedhenry-templates/fh-csharp-window-10-blank-app.git",
+        "githubUrl": "https://github.com:feedhenry-templates/fh-csharp-window-10-blank-app.git",
```
Comment (jasonmadigan) [inline global.json]: missing `/` after `github.com`

**Comment by jasonmadigan** [comment]
@edewit yeah, it's dumb - I'm guessing we've not hit it because we usually pop screenshots into the template's repo, merge that, and reference those images instead. Feel free to patch this out/continue on warning in `tasks/download.js`

**Comment by edewit** [comment]
@jasonmadigan ready to merge?

**Pull request merged**